gateleen-kafka/src/main/java/org/swisspush/gateleen/kafka/KafkaMessageSender.java

```diff
@@ -19,6 +19,7 @@
         List<Future<Void>> futureList = new ArrayList<>();
 
         for (KafkaProducerRecord<String, String> message : messages) {
+            log.debug("Start processing {} messages for kafka", messages.size());
```

Comment:
Please NOT inside the loop. This will print exactly the same log message multiple times.